SRIOV-config/UNIV/full-install.sh

```
@@ -148,10 +148,24 @@ function add_mc_realloc {
     fi
 }
 
-if [[ ${NIC_MODEL} == CX* ]]; then
-   echo "next, add add_mc_realloc"
-   prompt_continue
-   add_mc_realloc
+# Probe whether the NIC can allocate VFs without pci=realloc.
+WORKER_ARR=(${WORKER_LIST})
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)